### PR TITLE
Create Readme

### PR DESCRIPTION
**Pull request by ajwl**
This creates a readme for this project, covering installation, compilation and publishing to NPM.

### CODE REVIEW

**Comment by ajwl** [comment]
@regiskuckaertz hey. How does this work for you?
I haven't included information on deployment bcs I'm not sure how that works. I'm guessing it's the same story - publish to NPM and then update the dependency in Frontend?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+1. Build the JS code: `yarn build`
+
+2. Build the Scala code: `sbt run` 
+
+## Preview your changes
+
+In future we may add a dummy atom so the results of your change can be previewed from within the repo. 
+
+For now, you need to publish your changes to NPM as a new version of the Atom Renderer package. Then, you need to open
```
Comment (regiskuckaertz) [inline README.md]: Please specify that the semver rules for test packages should be followed here.

Comment (regiskuckaertz) [reply README.md]: Scratch that, just saw you did it a few lines below

**Comment by regiskuckaertz** [comment]
Thank you very much for having the impetus to write that

File: README.md (hunk 1 above)
Comment (ajwl) [reply README.md]: ah sure ;)

**Comment by ajwl** [comment]
@regiskuckaertz no worries ;) can I merge?

**Comment by regiskuckaertz** [comment]
oui

**Pull request merged**